uat/custom-components/client-java-paho/src/main/java/com/aws/greengrass/testing/mqtt311/client/paho/Mqtt311ConnectionImpl.java

```diff
@@ -127,6 +127,7 @@ public MqttPublishReply publish(long timeout, @NonNull Message message) {
         checkUserProperties(message.getUserProperties());
         checkContentType(message.getContentType());
```

Comment:
All content type, payload format indicator and message expiry interval are optional field. 
Default value of field is not a the same as abscence of optional field.
So, should pass result of message.hasContentType(),  message.hasPayloadFormatIndicator() and message.hasMessageExpiryInterval() to the checkAAA() methods.

Reply:
all the checks were used in GRPCControlServer